### ISSUE TITLE
Error: Hyperdrive is not a constructor

### ISSUE THREAD

**Issue by ninabreznik** [datproject/sdk#4]
Hey,

I am trying to run the example code from the readme and get this error. There was also a closing single quote typo (did a PR for that), but now it throws this error.

Any idea what this could be?

![Screenshot_2019-07-17_00-20-08](https://user-images.githubusercontent.com/1076427/61336545-9f39c680-a829-11e9-9189-cb37d46b0381.png)

**Comment by martinheidegger** [comment]
Hyperdrive - or hypercore for that matter - does not provide a constructor! 

https://github.com/datproject/sdk/blob/15d85c96e29bc74716ec8904c8eb051f9afd7222/index.js#L80-L88

@RangerMauve is that good? The documentation certainly will need updating?

**Comment by RangerMauve** [comment]
Whoops! It should be working with or without new. I'll fix it next week since I'm about to go off to dweb camp. 😁

Try using it without `new` for now. Sorry!

**Comment by ninabreznik** [comment]
Have fun at the dweb. Say hi to everyone (Karissa, MixMix etc.)

**Comment by RangerMauve** [comment]
Updated the docs to account for this.

tl;dr Don't use `new`!
